example_4_word_cloud/main.py

```diff
@@ -0,0 +1,34 @@
+import os
+from collections import Counter
+
+from wordcloud import WordCloud
+import matplotlib.pyplot as plt
+from speakleash import Speakleash
+import spacy
+
+base_dir = os.path.join(os.path.dirname(__file__))
+replicate_to = os.path.join(base_dir, "datasets")
+
+sl = Speakleash(replicate_to)
+wc = WordCloud()
+nlp = spacy.load('pl_core_news_md')
```

Comment:
I think it's safe to add `nlp.max_length` value to avoid memory allocation error, for example:

`nlp.max_length = 200_000`